src/models/image-map.ts

```diff
@@ -469,12 +466,12 @@ export const localAssetsImagesHandler: IImageHandler = {
     // We also need to convert legacy drawing tool stamp paths
     _url = _url.replace("assets/tools/drawing-tool/stamps", "msa/stamps");
 
-    if (this.imageMap.curriculumUrl) {
-      const imgUrl = new URL(_url, this.imageMap.curriculumUrl);
-      return { contentUrl: _url, displayUrl: imgUrl.href, success: true };
-    } else {
-      return { contentUrl: _url, displayUrl: _url, success: true };
-    }
+    // If curriculumUrl is defined and the image isn't in CLUE's own assets directory,
+    // build an absolute URL for the displayUrl.
+    const displayUrl = this.imageMap.curriculumUrl && !/^assets\//.test(_url)
+                         ? new URL(`../${_url}`, this.imageMap.unitUrl).href
```

Comment:
I think this would work, and is a little cleaner
```suggestion
                         ? new URL(_url, this.imageMap.curriculumUrl).href
```